weed/shell/command_fs_verify.go

```diff
@@ -245,8 +245,8 @@ func (c *commandFsVerify) verifyEntry(path string, chunks []*filer_pb.FileChunk,
 					defer wg.Done()
 					if err := c.verifyChunk(volumeServer, fChunk.Fid); err != nil {
 						if !(*c.metadataFromLog && strings.HasSuffix(err.Error(), "not found")) {
-							fmt.Fprintf(c.writer, "%s failed verify fileId %s: %+v\n",
-								msg, fChunk.GetFileIdString(), err)
+							fmt.Fprintf(c.writer, "%s failed verify fileId %s: %+v, at volume server %v\n",
+								fileMsg, chunk.GetFileIdString(), err, volumeServer)
```

Comment:
No need to change to `fileMsg, chunk`

Reply:
Fixed. Sorry for not catching it up myself. The code looked quite duplicate, so I just copy pasted it without giving it a second thought.